server/server/model-config.json

```diff
@@ -14,33 +14,49 @@
     ]
   },
   "User": {
-    "dataSource": "db"
+    "dataSource": "uniteMongoDB"
```

Comment:
Is it necessary to define the datasource for each model config ? Changing it in the future will be hard with this approach.

Reply:
When we create loopback model then it auto-generate this. If we did some changes and try to create some other model using loopback CLI then it overrides the file.